src/config/custom.js

```diff
@@ -1,5 +1,5 @@
 'use strict'
 
 module.exports = {
-  timeout: 80 * 1000
+  timeout: 40 * 1000
```

Comment:
I think that is a symptom of us not having something to remind us that certain operations should be fast and/or that tests should be a unit and not a massive set of operations.